test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -915,4 +915,14 @@ foo``u{2195}abc
         # Issue #2780
         { ExecuteCommand "`$herestr=@`"`n'`"'`n`"@" } | Should Not Throw
     }
+
+    It "Throw better error message when statement should be put in named blocks" {
+        try {
```

Comment:
Instead of try and catch we can use: 
`$err = { <command> } | Should -Throw -PassThru
$err.Exception.InnerException.ErrorRecord.FullyQualifiedErrorId | Should -BeExactly "MissingEndCurlyBraceOrNamedBlocks" `

Reply:
Will update to use the Pester V4 syntax.